components/Header/index.js

```diff
@@ -11,4 +11,27 @@
 // Use your function to create a header
 // and append it to the DOM inside the div.header-container
 
-function Header() {}
+function Header(date,title,temp) {
+    let header0 = document.createElement('div');
```

Comment:
I would recommend using const here because you aren't re-assigning any of these objects, you are only changing their attribute values. 

Not something you are expected to know but it's good coding practice!

Reply:
I've developed a bad habit of just using let. I'll practice being cognizant of when to use each.